qwp_integration_test.go

```
@@ -0,0 +1,712 @@
+	defer qwpDropTable(t, tableName)
+
+	s, err := newQwpLineSender(ctx, "ws://"+qwpTestAddr, qwpTransportOpts{}, time.Second, 0, 0, nil)
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer s.Close(ctx)
+
+	ts := time.Date(2024, 6, 15, 12, 0, 0, 0, time.UTC)
+
+	// Flush 1: symbols AAPL, MSFT.
+	s.Table(tableName).Symbol("sym", "AAPL").Int64Column("v", 1).At(ctx, ts)
```

golangci-lint (2.11.4)

[error] Error return value of `(github.com/questdb/go-questdb-client/v4.LineSender).At` is not checked

(errcheck)